core/src/wallet/cosmos/transaction_builders/CosmosLikeTransactionBuilder.cpp

```diff
@@ -648,6 +621,29 @@ namespace ledger {
                 tx->setMessages(messages);
             }
 
+            // FIXME Fix signature management
+            if (isSigned
+                &&  document.HasMember(kSignatures)
+                &&  document[kSignatures].IsArray()
+                && !document[kSignatures].GetArray().Empty()) {
+
+                auto firstSignature = document[kSignatures].GetArray()[0].GetObject();
+
+                { // Signature
+                    std::string sig = getString(firstSignature, kSignature);
+                    std::vector<uint8_t> sigBytes(sig.begin(), sig.end()); // FIXME Not the right way to do it...
```

Comment:
Also, I couldn't manage signatures properly yet...
I still need to understand how to convert from raw signature in json to binary data with correct representation.
If you have any input on this?...

Reply:
Maybe just sending the hexstring works : https://docs.tendermint.com/master/spec/blockchain/encoding.html#secp256k1 we'll have to test I suppose

Otherwise, the payload on the swagger looks like a base64 encoding, so maybe we can try that (b64Encode the `r || s` byte array) too

Reply:
If it's not b64 but expect an amino encoded value, then it's probably the length of the array and then `r||s` as stated here https://docs.tendermint.com/master/spec/blockchain/encoding.html

Reply:
Will try, thanks for the hints!